pkg/cmd/cli/cmd/login/helpers.go

```diff
@@ -80,10 +81,23 @@ func dialToServer(clientConfig restclient.Config) error {
 	return nil
 }
 
-func promptForInsecureTLS(reader io.Reader, out io.Writer) bool {
+func promptForInsecureTLS(reader io.Reader, out io.Writer, err error) bool {
+	var insecureTLSRequestReason string
```

Comment:
Don't think so, we are already filtered on the errors we check. If in future this changes or gets called by something else, you are asking for "prompt for insecure" when calling this anyway, and the reason line will just be suppressed.